draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -120,15 +120,16 @@ $CATZ
 
 # Description {#description}
 
-A catalog zone is a specially crafted DNS zone that contains, as DNS zone data:
+A catalog zone is a specially crafted DNS zone that contains, as DNS zone content:
 
-* A list of DNS zones (called "member zones").
+* A list of DNS zones (called "member zones"), plus properties associated with those zones.
 
 Implementations of catalog zones SHOULD ignore any RR in the catalog zone which is meaningless or useless to the implementation.
 
 Authoritative servers may be preconfigured with multiple catalog zones, each associated with a different set of configurations.
 A member zone can as such be reconfigured with a different set of preconfigured settings by removing it as a member of one catalog zone and making it a member of another.
 
+[FIXME: what is the point? How would the secondary know that a member zone is another catalog zone?]
```

Comment:
I was thinking of preconfigured properties for catalog zones when they are present.
This would enable activating secondaries for groups of zones at once.
I now think this is too complex to explain and far fetched and suggest to just remove line 133.

Reply:
:+1: